templates/registration/register.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Register{% endblock %}
+
+{% block stylesheets %}
+    {{ encore_entry_link_tags('login_style') }}
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <div class="row justify-content-center sign-container">
+            <div class="col-xs-12 col-sm-6 col-md-4">
+                <h1 class="text-center login-title">Registration</h1>
+                <div class="account-wall">
+                    <img class="profile-img" src="{{ asset('build/images/login/add-user-icon-256.png') }}"
+                         alt="">
+
+                    {{ form_start(registrationForm, {'attr': {'class': 'form-signin'}}) }}
```

Comment:
Registration also known as `sign up`, not `sign in`. :)

Reply:
Resolve